src/Result/PersonalityStyleInfo.js

```diff
@@ -2,49 +2,64 @@ import React from 'react'
 import { useParams } from 'react-router-dom'
 import { personalityStyleData } from '../data/personalityStyleData'
 import { capitalizeFirstLetter } from '../common/util'
+import HeadlineUnderline from '../common/HeadlineUnderline'
+import styled from 'styled-components'
+import SectionBG from '../common/SectionBG'
+import BulletList from '../common/BulletList'
 
 export default function PersonalityStyleInfo() {
   const { styleId } = useParams()
   const currentStyleData = personalityStyleData[styleId - 1]
 
   return (
     <>
-      <h1>{capitalizeFirstLetter(currentStyleData.name)}</h1>
-      <h2>Einleitung</h2>
-      <p>{currentStyleData.description}</p>
-      <h2>Stärken</h2>
-      <ul>
-        {currentStyleData.strengths.map((strength) => {
-          return <li key={strength}>{strength}</li>
-        })}
-      </ul>
-      <h2>Schwächen</h2>
-      <ul>
-        {currentStyleData.weaknesses.map((weakness) => {
-          return <li key={weakness}>{weakness}</li>
-        })}
-      </ul>
-      <h2>Tipps</h2>
-      <ul>
-        {currentStyleData.tips.map((tip) => {
-          return <li key={tip}>{tip}</li>
-        })}
-      </ul>
+      <HeadlineUnderline>
+        <h1>{capitalizeFirstLetter(currentStyleData.name)}</h1>
+      </HeadlineUnderline>
+      <SectionNoBG>
+        <HeadlineNoMarginTop>Einleitung</HeadlineNoMarginTop>
+        <p>{currentStyleData.description}</p>
+      </SectionNoBG>
+      <InfoBG>
+        <h2>Stärken</h2>
+        <BulletList listType="bullet_circle">
+          {currentStyleData.strengths.map((strength) => {
+            return <li key={strength}>{strength}</li>
+          })}
+        </BulletList>
+      </InfoBG>
+      <SectionNoBG>
+        <h2>Schwächen</h2>
+        <BulletList listType="bullet_circle">
+          {currentStyleData.weaknesses.map((weakness) => {
+            return <li key={weakness}>{weakness}</li>
+          })}
+        </BulletList>
+      </SectionNoBG>
+      <InfoBG>
+        <h2>Tipps</h2>
+        <BulletList listType="bullet_circle">
+          {currentStyleData.tips.map((tip) => {
+            return <li key={tip}>{tip}</li>
+          })}
+        </BulletList>
+      </InfoBG>
       <h2>Weitere Informationen</h2>
-      <ul>
+      <BulletList listType="bullet_circle">
         {currentStyleData.furtherInfo.map((info, index) => {
           return (
             <li key={index}>
```

Comment:
I might marked this one before, but when mapping over an array, one should go for the ID as key values. Not the index position.